src/panels/detail/components/connections/ConnectionCache.tsx

```diff
@@ -0,0 +1,133 @@
+import React from "react";
+import { AppSettings, WorkspaceLinks } from "../../../../config/Variables";
+import { getLabelOrBlank } from "../../../../function/FunctionGetVars";
+import { getVocabularyShortLabel } from "@opendata-mvcr/assembly-line-shared";
+import { Badge, OverlayTrigger, Popover } from "react-bootstrap";
+import { CacheSearchVocabularies } from "../../../../datatypes/CacheSearchResults";
+import _ from "underscore";
+import { getOtherConnectionElementID } from "../../../../function/FunctionLink";
+import { Representation } from "../../../../config/Enum";
+import Connection from "./Connection";
+
+interface Props {
+  connection: {
```

Comment:
This type is just copied from another file... should be definitely extracted to a separate external type so that it can be reused.